skimage/restoration/_rolling_ball.py

```diff
@@ -38,20 +39,23 @@ def rolling_ball(image, *, radius=100, kernel=None, nansafe=False, num_threads=N
 
     Notes
     -----
-    For the pixel that has its background intensity estimated (without loss
-    of generality at ``center``) the rolling ball method centers ``kernel``
-    under it and raises the kernel until the surface touches the image umbra
-    at some ``pos=(y,x)``. The background intensity is then estimated
-    using the image intensity at that position (``image[pos]``) plus the
-    difference of ``kernel[center] - kernel[pos]``.
+    The algorithm is easy to grasp in 2D: Consider that each pixel value
+    defines a height, forming a 2D surface in 3D space. Then, a (3D) ball of
+    given radius (or a kernel, in the general case) is placed under this
+    surface and raised until it touches it. The background intensity is
+    estimated by the hull of the volume reachable by the ball.
 
-    This algorithm assumes that dark pixels correspond to the background. If
-    you have a bright background, invert the image before passing it to the
-    function, e.g., using `utils.invert`. See the gallery example for details.
+    Clearly, for this method to give meaningful results, the radius of the
```

Comment:
```suggestion
    For this method to give meaningful results, the radius of the
```

"Clearly" tends to make people bad. "Err, but I didn't see that!"

Reply:
I can see that. The adverb is a left-over from previous wording attempts, but basically I wanted to keep the reader in the 'visualization' of things we gave in the previous paragraph. Basically, I'd like to convey something like: "*You can (clearly) see* that if the ball is smaller that the features of interest, it will raise through the fine details and miss the background capture we intend..."